http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterp.scala

```diff
@@ -179,7 +182,7 @@ final class LeoAppServiceInterp[F[_]: Parallel](
               F.raiseError[Option[LastUsedApp]](
                 new LeoException("Existing disk found, but no restore info found in DB", traceId = Some(ctx.traceId))
               )
-            case (Some(FormattedBy.GCE), _) | (Some(FormattedBy.Custom), _) =>
+            case (Some(FormattedBy.GCE), _) | (Some(FormattedBy.Custom), _) | (Some(FormattedBy.Cromwell), Some(_)) =>
               F.raiseError[Option[LastUsedApp]](
                 new LeoException(
                   s"Disk is formatted by ${diskResult.disk.formattedBy.get} already, cannot be used for galaxy app",
```

Comment:
This error message shouldn't refer to Galaxy, should it? And what is this error case?

Reply:
Or maybe Galaxy is correct here because this code is specifically a Galaxy pathway?